shopinvader_api_lead/schemas/lead.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from extendable_pydantic import StrictExtendableBaseModel
+
+
+class Lead(StrictExtendableBaseModel):
+    id: int
```

Comment:
Add the missing fields

Reply:
I propose to make a new PR to add these missing fields

Reply:
Ok for me